docs/performance/proposal.md

```diff
@@ -0,0 +1,69 @@
+# Measuring Analyzer Performance
+
+Now that analyzer are part of the build we need a mechanism to track their performance across releases as well as build confidence regarding their use in the SDK.
+
+## Goals
+
+- Developers can quickly get feedback on how their change effects performance
+- We can track and detect performance regressions in builds before release.
+
+## What we do today
+
+- Roslyn
+  - Can be run on CI: **No**
+  - Can be run locally with a single script: **No**
```

Comment:
😕 I don't understand this

Reply:
where is the script that runs your changes against the what is checked into main? I must have missed it